packages/react-components/src/components/ChatMessage/MyMessageComponents/FluentChatMyMessageComponent.tsx

```diff
@@ -190,7 +197,8 @@ export const FluentChatMyMessageComponent = (props: FluentChatMessageComponentWr
 
   const myMessageBodyProps = useMemo(() => {
     return {
-      className: mergeClasses(chatMessageRenderStyles.bodyCommon, chatMessageRenderStyles.bodyMyMessage)
+      className: mergeClasses(chatMessageRenderStyles.bodyCommon, chatMessageRenderStyles.bodyMyMessage),
+      ref: bodyRef
```

Comment:
nit: should we make this name more specific, such as bodyMyMessageRef?

Reply:
I though about it too but we have only components for my message component in this file (it's part of the FluentChat`My`MessageComponent)...

Reply:
Updated to fluentMessageBodyRef